services/motion/builtin/move_request.go

```diff
@@ -343,17 +343,27 @@ func validateNotNegNorNaN(f float64, name string) error {
 	return validateNotNeg(f, name)
 }
 
-func newValidatedMotionCfg(motionCfg *motion.MotionConfiguration) (*validatedMotionConfiguration, error) {
+func newValidatedMotionCfg(motionCfg *motion.MotionConfiguration, reqType requestType) (*validatedMotionConfiguration, error) {
 	empty := &validatedMotionConfiguration{}
 	vmc := &validatedMotionConfiguration{
 		angularDegsPerSec:     defaultAngularDegsPerSec,
 		linearMPerSec:         defaultLinearMPerSec,
 		obstaclePollingFreqHz: defaultObstaclePollingHz,
 		positionPollingFreqHz: defaultPositionPollingHz,
-		planDeviationMM:       defaultPlanDeviationM * 1e3,
 		obstacleDetectors:     []motion.ObstacleDetectorName{},
 	}
 
+	switch reqType {
+	case requestTypeMoveOnGlobe:
+		vmc.planDeviationMM = defaultGlobePlanDeviationM * 1e3
```

Comment:
This overwrites anything that was in the motionCfg

Reply:
i disagree, below we do
```
if motionCfg.PlanDeviationMM != 0 {
		vmc.planDeviationMM = motionCfg.PlanDeviationMM
	}
```

Reply:
Ah I didnt see that